doc/index.rst

```diff
@@ -42,43 +44,89 @@ game without learning a complex framework.
     <p><a href="https://vimeo.com/662273899">Quick Arcade Library Introduction Video</a></p>
    </div>
 
-The library was originally designed to be used in an educational setting, but
-it has evolved to be a powerful library for creating games. It is used in
-education, prototyping, and game jams. Hobbyists wanting to create a game
-quickly can benefit from the library's simple API.
-
+Open Source, Libre Assets
```

Comment:
i assume it should be 'Library'

Reply:
Nope, it's a term. It's distinct from copyleft and the shortest word I could find. I'll think about this heading, but I may merge it as-is for the time being.

Reply:
Nope, ["libre" is a term](https://en.wikipedia.org/wiki/Free_software), but the fact you're confused means I should replace it.

Reply:
Update: rephrased the title.